stm/stm.ml

```diff
@@ -1732,7 +1737,7 @@ end = struct (* {{{ *)
     | `ERROR -> exit 1
     | `ERROR_ADMITTED -> cst, false
     | `OK_ADMITTED -> cst, false
-    | `OK (po,_) ->
+    | `OK po ->
```

Comment:
BTW it's interesting that we only use the `name` field, maybe we should do
```suggestion
    | `OK {po.Proof_global.name=id} ->
```
instead? Or even project when returning the `` `OK `` in `check_task_aux`.

Reply:
Done with a record projection, which seems less invasive to me.